src/language-js/embed.js

```diff
@@ -165,12 +168,24 @@ function embed(path, print, textToDoc /*, options */) {
       break;
     }
   }
+
+  function printMarkdown(text) {
+    const doc = textToDoc(text, { parser: "markdown", __inJsTemplate: true });
+    return docUtils.stripTrailingHardline(escapeBackticks(doc));
+  }
+}
+
+function literalify(doc) {
+  return align(-Infinity, doc);
+}
```

Comment:
Could you make this a “primitive” (like how `conditionalGroup` is actually a `group`) in the doc file?

Reply:
Not sure how should we name it, `rootIndent`?

Reply:
Sure. Or maybe `dedentToRoot`?